db/database.py

```
@@ -0,0 +1,537 @@
+    mitmb = MitmBundle.model_validate(fake_pre_keys)
+    print(mitmb)
+    print(mitmb.model_dump_json(indent=2, by_alias=True))
+    print(mitmb.fake_kyber_keys)
+
+    with DatabaseSessionManager().get_session() as session:
+        session.merge(mitmb)
+        session.commit()
+
+    with DatabaseSessionManager().get_session() as session:
+        print("FRESH FROM DB! (MitmBundle)")
+        meep: MitmBundle = ses.exec(select(MitmBundle)).first()
```

Ruff

Undefined name `MitmBundle`

(F821)